docs/development/api/changes.md

```diff
@@ -1,5 +1,9 @@
 # API Changes
 
+## 4.x vs 3.x
+
+- The Animal asset `is_castrated` attribute was renamed to `is_sterile`.
```

Comment:
Not sure it affects this PR, but it occurs to me that `is_sterile` could also be a useful attribute for plant/seed/compost assets - i.e. triploid, non-pollen-producing, infertile hybrid, hot composted, etc 🤓.

Reply:
True! It wouldn't be too hard to add it to other asset types. Just a matter of deciding which. Plant and Seed make sense... you also mentioned Compost but I think the "sterile" meaning is slightly different there (aka `is_sterilized` or `is_kept_sterile`). I remember talking about this on a call together a while back...

We could also defer to the future "log based attributes" work on this...

Reply:
Agreed on deferring this.

I guess you're also right about compost maybe being a different case. Modeling levels of heterogeneity, contamination, fecundity over time all seem related though.